packages/react-vapor/src/components/modal/ModalComposite.tsx

```diff
@@ -107,7 +109,7 @@ export class ModalComposite extends React.PureComponent<
                 onAfterClose={this.props.closeCallback}
                 {...reactModalprops}
             >
-                <div className="modal-content" id={this.props.id}>
+                <div className={classNames('modal-content', this.props.contentClasses)} id={this.props.id}>

```

Comment:
Can you add a unit test?

Reply:
I am curious what is the use-case behind this

Reply:
Its for the height of the modal content. This is actually only accessible by using the ID of the Modal.

I can add an unit test.